CMakeLists.txt

```diff
@@ -240,29 +240,26 @@ if(LMMS_BUILD_APPLE)
 endif()
 find_package(Perl)
 
-IF(WANT_LV2)
-	IF(PKG_CONFIG_FOUND)
```

Comment:
I know these guards scattered everywhere are ugly, but did you mean to remove it?

Reply:
Fixed